identities/views.py

```diff
@@ -257,6 +258,23 @@ def perform_create(self, serializer):
         return serializer.save(created_by=self.request.user)
 
 
+class OptOutSearchList(generics.ListAPIView):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = OptOutSerializer
+
+    def get_queryset(self):
+        query_params = list(self.request.query_params.keys())
+        filter_criteria = {}
+
+        for filter in query_params:
+            filter_criteria[filter] = self.request.query_params[filter]
```

Comment:
It is actually converting it from a QueryDict to Dict, if I use dict() to convert it I don't get the same results.